cyences_app_for_splunk/default/data/ui/views/cs_gws_reports.xml

```diff
@@ -529,4 +529,55 @@
       </table>
     </panel>
   </row>
+  <row>
+    <panel>
+      <title>Google Drive objects shared Outside or with External User</title>
+      <table>
+        <search>
+          <query>`cs_gws` sourcetype="gws:reports:drive" "event.type"=acl_change visibility IN ("shared_externally","people_with_link")
+| rex field=owner "[^@]+@(?&lt;src_domain&gt;[^@]+)" 
+| rex field=email "[^@]+@(?&lt;dest_domain&gt;[^@]+)" 
+| where isnotnull(src_domain) AND isnotnull(dest_domain)
+| search src_domain IN `cs_email_user_domain` AND NOT dest_domain IN `cs_email_user_domain`
+| `cs_user_privilege_mapping(owner)` 
+| stats values(doc_title) as doc_title, values(doc_type) as doc_type, values(event.name) as event.name, values(email) as dst_email_list, values(visibility) as visibility, count, min(_time) as firstTime, max(_time) as lastTime by owner
+| eval cyences_severity = if(is_privileged_user=="Yes", "critical", "medium") 
+| `cs_human_readable_time_format(firstTime)` 
+| `cs_human_readable_time_format(lastTime)`
+| `cs_gws_google_drive_objects_shared_externally_filter`</query>
```

Comment:
We already defined the macro: **cs_gws_drive_object_shared_outside_filter** use this.